### PR TITLE
[ACS-7449] Selected filter styles broken

### PR DESCRIPTION
**Pull request by tomson7777**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Fix broken styles of filters after migration

**What is the new behaviour?**
Filter chip element is correctly displayed

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

### CODE REVIEW

File: lib/content-services/src/lib/search/components/search-filter-chips/search-filter-chips.component.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ import { FacetField, SearchCategory, TabbedFacetField } from '../../models';
     encapsulation: ViewEncapsulation.None
 })
 export class SearchFilterChipsComponent implements OnInit, OnDestroy {
-    private queryBuilder = inject(SearchQueryBuilderService);
+    public queryBuilder = inject(SearchQueryBuilderService);
```
[the remark this answers is not in the document]
Comment (MichalKinas) [reply lib/content-services/src/lib/search/components/search-filter-chips/search-filter-chips.component.ts:32]: I think this is a some sort of rebase issue, this works correctly on develop with `get categories()` which is then used in html template, please check how it's done on develop and let's use the same solution here

File: lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.ts [hunk 2]
```diff
@@ -22,19 +22,26 @@ import { MatMenuTrigger } from '@angular/material/menu';
...
-export class SearchWidgetChipComponent  {
+    selector: 'adf-search-widget-chip',
+    templateUrl: './search-widget-chip.component.html',
+    styles: [
+        `
+            .adf-search-widget-extra-width {
+                max-width: 500px;
+            }
+
+            .mdc-evolution-chip__checkmark {
```
[the remark this answers is not in the document]
Comment (MichalKinas) [reply lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.ts:33]: Okay so what Jacek ment is to have a file where material selectors will be mapped and not actually styled e.g. `$mat-table: '.mat-mdc-table';` So in my eyes we can have something similar in ACA to satisfy the stylelint, as for the styling than for sure I agree that we should keep most of the styling per component, the only exception should be some overrides for component that can be made global

File: lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.ts [hunk 3]
```diff
@@ -22,19 +22,26 @@ import { MatMenuTrigger } from '@angular/material/menu';
...
+                display: none;
+            }
 
+            .mat-mdc-chip-graphic {
+                width: 0 !important;
```
Comment (nikita-web-ua) [inline lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.ts:38]: I think we can avoid `!important` by using more precise selector

**Comment by MichalKinas** [comment]
This issue has already been fixed by other PR hence closing this one

**Pull request closed without merging**